components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -193,8 +203,19 @@ - (CGFloat)clearButtonSideLengthWithTextFieldSize:(CGSize)textFieldSize {
   return systemPlaceholderRect.size.height;
 }
 
+- (CGSize)preferredSizeWithWidth:(CGFloat)width {
+  CGSize fittingSize = CGSizeMake(width, CGFLOAT_MAX);
+  MDCBaseTextFieldLayout *layout = [self calculateLayoutWithTextFieldSize:fittingSize];
+  return CGSizeMake(width, layout.calculatedHeight);
+}
+
 #pragma mark UITextField Accessor Overrides
 
+- (void)setEnabled:(BOOL)enabled {
+  [super setEnabled:enabled];
```

Comment:
new line after call to super

Reply:
I couldn't find the part of the Objective-C style guide that says this is a requirement. In fact, every code example of a different rule in the style guide that I saw that happened to have call to super in it did _not_ have an empty line under it.

Reply:
Our code base skews primarily toward having an empty newline after the call to super; please continue to lean into that convention barring a strong stance from the style guide otherwise.

Newline after super call:

`^ +\[super.*\];\n *\n`: 550 results

A line that starts with a character, digit, or `[` after super call:

`^ +\[super.*\];\n +[\w\d\[]+`: 206 results

Reply:
Okie dokie